highway_env/vehicle/graphics.py

```diff
@@ -27,27 +27,30 @@ def display(cls, vehicle, surface, transparent=False):
         :param vehicle: the vehicle to be drawn
         :param surface: the surface to draw the vehicle on
         :param transparent: whether the vehicle should be drawn slightly transparent
+        :param offscreen: whether the rendering should be done offscreen or not
         """
         v = vehicle
         s = pygame.Surface((surface.pix(v.LENGTH), surface.pix(v.LENGTH)), pygame.SRCALPHA)  # per-pixel alpha
         rect = (0, surface.pix(v.LENGTH) / 2 - surface.pix(v.WIDTH) / 2, surface.pix(v.LENGTH), surface.pix(v.WIDTH))
         pygame.draw.rect(s, cls.get_color(v, transparent), rect, 0)
         pygame.draw.rect(s, cls.BLACK, rect, 1)
-        s = pygame.Surface.convert_alpha(s)
+        if not offscreen:  # convert_alpha throws errors in offscreen mode TODO() Explain why
+            s = pygame.Surface.convert_alpha(s)
```

Comment:
Oh that's annoying. Maybe wrapping this line in a try/except would have been easier than passing the `offscreen` variable all the way here?